.goreleaser.yml

```diff
@@ -29,6 +29,12 @@ builds:
   ignore:
     - goos: darwin
       goarch: '386'
+    - goos: darwin
+      goarch: 'arm'
```

Comment:
Nit: `'386'` needs to be quoted but the other values don't.

Reply:
done.